llvm/include/llvm/BinaryFormat/COFF.h

```diff
@@ -806,6 +806,12 @@ enum Feat00Flags : uint32_t {
   Kernel = 0x40000000,
 };
 
+enum class Arm64ECThunkType : uint32_t {
```

Comment:
Should the underlying type for the enum be `uint8_t`, since that's what we end up casting it to?

Reply:
I changed it, thanks. I wasn't sure about that, it occupies 32 bits in `.hybmp` entry, but only 8 of them are valid.